main/inc/lib/sessionmanager.lib.php

```diff
@@ -5430,4 +5430,87 @@ public static function getCoursesListByCourseCoach($coachId)
         ));
     }
 
+    /**
+     * Get a session list filtered by name, description or short description extra field
```

Comment:
This should be updated now, given that "short description" is just a parameter in the array